src/content/reference/gsctl/list-clusters.md

```diff
@@ -30,12 +30,12 @@ The details displayed are:
 
 ## Argument reference
 
-- `--output` or `-o`: Using this flag with the value `json`, the output can be printed in JSON format. This is convenient for use in automation. JSON representation additionally contains user defined cluster labels for clusters with node pools on AWS. The default output format is `table`, which results in an output like shown above.
+- `--output` or `-o`: Using this flag with the value `json`, the output can be printed in JSON format. This is convenient for use in automation. JSON representation additionally contains user defined cluster labels for clusters with release version 10.0.0 and above on AWS. The default output format is `table`, which results in an output like shown above.
```

Comment:
```suggestion
- `--output` or `-o`: Using this flag with the value `json`, the output can be printed in JSON format. This is convenient for use in automation. JSON representation additionally contains user defined cluster labels for clusters with release version {{% first_aws_nodepools_version %}} and above on AWS. The default output format is `table`, which results in an output like shown above.
```

I just recalled that we have a shortcode to avoid duplicating this bit of information.

Reply:
Very nice! Adding this :+1: